### DISCUSSION TITLE
마일스톤 활용 방안

### DISCUSSION THREAD

**Discussion by Dae-Hwa** [Sc0up/scoup-backend#125] [Convention]
구현이 단계별로 진행될 경우 각 유스케이스 별로 여러개의 이슈가 발생하게 됩니다.
제인이 구현하던 파트를 예시로 생각해보면 될 것 같아요.
이 경우 크게 보면 하나의 기능 구현인데 세부 기능때문에 이슈가 흩어지게 됩니다.
따라서 하나의 유스케이스를 마일스톤으로 잡고 내부의 기능을 처리하는 이슈를 만들어 처리하는 방안을 생각해봤습니다.
의견 부탁드려요~!

**Comment by janeljs** [comment]
앗 2일 전에 올린 디스커션이었군요! 확인을 이제 했네요ㅎㅎ
처음에는 너무 작은 단위의 마일스톤이 아닌가 했었는데 사용하다보니 뿌듯하고 기분도 좋고, 노션보다 더 관리하기도 편한 것 같습니다~

**Reply by janeljs** [reply]
![image](https://user-images.githubusercontent.com/68000537/132218207-058110d5-92c8-4655-96f9-ff06b621675d.png)
짠짠! 저는 벌써 적용했어요ㅋㅋ
